src/SourceBuild/content/Directory.Build.props

```diff
@@ -92,6 +92,9 @@
     <ArtifactsObjDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'obj'))</ArtifactsObjDir>
     <ArtifactsBinDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'bin'))</ArtifactsBinDir>
     <ArtifactsLogDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'log', '$(Configuration)'))</ArtifactsLogDir>
+    <ArtifactsPackagesDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'packages', '$(Configuration)'))</ArtifactsPackagesDir>
```

Comment:
I find it a little confusing that this is called packages when is contains more than just nupkgs.  I know this is the arcade way but...

Reply:
Agreed, but I'd rather stay consistent with all other arcadified repos.

Reply:
Technically all other repos don't use that same layout. I wuld stick with packages == nuget packages and a sibling 'assets' == everything else.

Reply:
Chatted with Matt offline. I will move the assets out.